Formula/apophenia.rb

```diff
@@ -0,0 +1,31 @@
+class Apophenia < Formula
+  desc "C library for statistical and scientific computing"
+  homepage "http://apophenia.info/"
+  url "https://github.com/b-k/apophenia/archive/v1.0.tar.gz"
+  sha256 "c753047a9230f9d9e105541f671c4961dc7998f4402972424e591404f33b82ca"
+
+  depends_on "gsl"
+  depends_on "sqlite"
```

Comment:
qpophenia needs the sqlite library, not just the command line binary.  but I can remove the depends_on.